components/Flashcard.js

```diff
@@ -113,6 +131,14 @@ export default function Flashcard({
         >
           delete
         </button>
+        <StyledLink
+          href={`/edit?id=${flashcard.id}`}
```

Comment:
please use dynamic routes from next.js for passing the card id instead. You have more documentation about it in our handouts and on the next.js website.
```suggestion
          href={`flashcards/${flashcard.id}/edit`}
```
Please make sure to move the edit.js page into a new folder folder `/pages/flashcards/[cardId]/edit.js`

Reply:
Done